cmd/osmosisd/cmd/testnetify/state_export_to_testnet.go

```diff
@@ -44,8 +43,6 @@ func defaultTestnetParams() TestnetParams {
 	consensusPubkey := "2OpBuqaXvXQ+lSxAoT1S7Jfyr56KiakTzvuFiuJK+X4="
 	// mnemonic kitchen comic flower drip sick prize account cheese truth income weekend nominee segment punch call satisfy captain earth ethics wasp clump tunnel orchard advance
 	operatorPubkey := "osmopub1addwnpepq0wv95fxk32z90u42t3df2l5pdtngvg20rkalv5vt2d7n5q5ekk35d8hh20"
-	operatorValOper := "osmovaloper1qye772qje88p7ggtzrvl9nxvty6dkuusvpqhac"
```

Comment:
unused

Reply:
lets undo this, and just add in _ = operatorValOper.

A lot of the point of this file is starter code to build off of